From 761923b8837f307b39a95ee53b5863485cf4cd43 Mon Sep 17 00:00:00 2001
From: Matt Coster <matt.coster@imgtec.com>
Date: Mon, 29 Jun 2026 16:47:29 +0100
Subject: [PATCH 01479/15333] MAINTAINERS, mailmap: Update address for Matt
 Coster

I'm leaving Imagination; remove myself as a maintainer but leave
a personal forwarding address.

Acked-by: Matt Coster <opensource@mtcoster.net>
Acked-by: Alessio Belle <alessio.belle@imgtec.com>
Link: https://patch.msgid.link/20260629-goodbye-v1-1-3bab53a80c53@imgtec.com
Signed-off-by: Matt Coster <matt.coster@imgtec.com>
---
 .mailmap    | 1 +
 MAINTAINERS | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/.mailmap b/.mailmap
index 23eb9a4b04f4..d0d1367a261b 100644
--- a/.mailmap
+++ b/.mailmap
@@ -558,6 +558,7 @@ Martyna Szapar-Mudlaw <martyna.szapar-mudlaw@linux.intel.com> <martyna.szapar-mu
 Mathieu Othacehe <othacehe@gnu.org> <m.othacehe@gmail.com>
 Mat Martineau <martineau@kernel.org> <mathew.j.martineau@linux.intel.com>
 Mat Martineau <martineau@kernel.org> <mathewm@codeaurora.org>
+Matt Coster <opensource@mtcoster.net> <matt.coster@imgtec.com>
 Matthew Wilcox <willy@infradead.org> <matthew.r.wilcox@intel.com>
 Matthew Wilcox <willy@infradead.org> <matthew@wil.cx>
 Matthew Wilcox <willy@infradead.org> <mawilcox@linuxonhyperv.com>
diff --git a/MAINTAINERS b/MAINTAINERS
index c1004533b00a..50f2aa8c0d8a 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -12670,7 +12670,6 @@ S:	Orphan
 F:	drivers/media/rc/img-ir/
 
 IMGTEC POWERVR DRM DRIVER
-M:	Matt Coster <matt.coster@imgtec.com>
 M:	Alessio Belle <alessio.belle@imgtec.com>
 M:	Luigi Santivetti <luigi.santivetti@imgtec.com>
 L:	imagination@lists.freedesktop.org
-- 
2.55.0

